packages/flutter_tools/lib/src/runner/flutter_command.dart

```diff
@@ -138,6 +138,9 @@ abstract class FlutterCommand extends Command<Null> {
   /// tracking of the command.
   Future<String> get usagePath async => name;
 
+  /// Additional usage values to be sent with the usage ping.
+  Future<Map<String, String>> get usageValues async => const <String, String>{};
```

Comment:
Is usageValue perhaps a bit vague? Since it's in FlutterCommandResult, we might still want to mention that it's analytics related. Perhaps this should replace the original name of 'analyticsParameter' or 'analyticsDimensions' or some such.